api/src/test/java/org/openmrs/api/impl/ConceptServiceImplTest.java

```diff
@@ -288,6 +288,64 @@ public void saveConcept_shouldForceSetFlagIfSetMembersExist() {
 		assertTrue(concept.getSet());
 	}
 	
+	/**
+	 * @see ConceptServiceImpl#saveDrug(Drug) 
+	 */
+	@Test
+	public void saveDrug_shouldPutGeneratedIdOntoReturnedDrug() {
+		Drug drug = new Drug();
+		Concept concept = new Concept();
```

Comment:
Instead of creating and saving a concept from scratch, you could alternatively just load an existing concept with `conceptService.getConcept(conceptId)` from the ones here which are automatically loaded in the test: https://github.com/openmrs/openmrs-core/blob/master/api/src/test/resources/org/openmrs/include/standardTestDataset.xml#L49-L84

But what you have done is till fine. So i will merge it anyway. 😊